plugins/aap-backend/package.json

```diff
@@ -53,7 +53,7 @@
     "@backstage/backend-test-utils": "1.0.0",
     "@backstage/cli": "0.27.1",
     "@backstage/plugin-catalog-backend": "1.26.1",
-    "@janus-idp/cli": "1.15.2",
+    "@janus-idp/cli": "*",
```

Comment:
Would this work alright on the `release-1.3` branch?  I think probably, as it'd be using that branch of the source.  However this would mean we should be careful to always ensure CLI changes cause CLI releases, otherwise it's possible this could do a release using CLI code that is on the main branch but unreleased.